packages/ui/src/styles/colors.ts

```diff
@@ -100,5 +100,6 @@ const _palettes = {
   },
 };
 
+const typedKeys = Object.keys as <T>(o: T) => Extract<keyof T, string>[];
```

Comment:
_FYI_: I decided to inline due to the only other usage of this function is within `packages/core`. That being said, it could be moved to `core2` if you expect other usages!